---
description: 'Literature, Course and Book Recommendations'
---

# Machine learning & AI

## 1. **Startup of Literature Reading**

* **Deep Learning**: 2015 _Nature Biotech._ - DeepBind: Predicting the sequence specificities of DNA- and RNA-binding proteins by deep learning
* **Cancer Locator**:  2017 _Genome Biology_ - CancerLocator: non-invasive cancer diagnosis and tissue-of-origin prediction using methylation profiles of cell-free DNA
* **Batch correction**: 2018 _Nature Biotech._ - Batch effects in single-cell RNA-sequencing data are corrected by matching mutual nearest neighbors

[PDFs](https://cloud.tsinghua.edu.cn/d/928f3f4a8c8d4ab8b8ad/?p=/Literature%20shared%20by%20John/Recommendation%20for%20Startup/AI%20%26%20Machine%20Learning&mode=list)

## 2. Deep Learning on RNA

> * **Course:**  Machine Learning \(CS229\) -- Andrew Ng : [coursera](https://www.coursera.org/learn/machine-learning)\*\*\*\*
> * **Book:    《**机器学习》-- 周志华 \([link](https://book.douban.com/subject/26708119/)\)  \([更多书籍课程推荐](https://lulab1.gitbook.io/training/appendix/appendix-i.keep-learning)\)

* **AS of RNA**: 2019 _Cell_ - Predicting Splicing from Primary Sequence with Deep Learning
* **AS of RNA \(DARTS**\): 2019 _Nature Methods_ - Deep-learning augmented RNA-seq analysis of transcript splicing
* **APA of RNA**: 2019 _Cell_ - A Deep Neural Network for Predicting and Engineering Alternative Polyadenylation
* **RNA/DNA-Protein Binding \(DeepBind\)**: 2015 _Nature Biotech._ - DeepBind: Predicting the sequence specificities of DNA- and RNA-binding proteins by deep learning
* **RNA secondary structure**: see [below](ai.md#3-1-rna-secondary-structure-prediction)
* **Cancer prediction using exRNA-seq**: see [below](ai.md#4-5-transfer-learning)

[PDFs](https://cloud.tsinghua.edu.cn/d/9553a9a553304ff7b311/?p=%2FDeep%20Learning%20on%20RNA&mode=list)

## 3. Comp. Methods for Liquid Biopsy

### 3.1 Imputation, Normalization and Batch Correction

* **Methods for Single Cell RNA-seq** 
  * \(1\) Dropout/Sparseness and Imputation
  * \(2\) Heterogeneity and Normalization
  * \(3\) Batch effect and Confounder
  * \(Pseudo-time and Others\)
* **Recommendations**:
  * 2020 _Nature COMMN_ - Embracing the **dropouts** in single-cell RNA-seq analysis
  * 2019 - _Nature Methods_ - A discriminative learning approach to differential expression analysis for single-cell RNA-seq
  * 2017 - _Nature Methods_ - Normalizing single-cell RNA sequencing data: challenges and opportunities

> [PDFs](https://cloud.tsinghua.edu.cn/d/9553a9a553304ff7b311/?p=%2Fmethod%20-%20single%20cell&mode=list)
>
> * \(1\) Dropout/Sparseness and Imputation
> * \(2\) Heterogeneity and Normalization
> * \(3\) Batch effect and Confounder
> * \(Pseudo-time and Others\)
>
> [Tutorial](https://lulab1.gitbook.io/training/part-iii.-case-studies/case-study-1.exrna-seq/1.4.normalization-issues)

### 3.2  Feature Selection Method

[PDFs](https://cloud.tsinghua.edu.cn/d/9553a9a553304ff7b311/?p=%2Fmethod%20-%20Feature%20Selection&mode=list)

### 3.3 Network Approach and Clustering

* 2020 _Bioinformatics_ - Deeptype - Deep-learning approach to identifying cancer subtypes using high-dimensional genomic data
* 2018 _Nature COMMN._ - Pathway based subnetworks enable cross-disease biomarker discovery

[PDFs](https://cloud.tsinghua.edu.cn/d/9553a9a553304ff7b311/?p=%2Fmethod%20-%20network%20and%20clustering&mode=list)

### 3.4 Tumor Location Method

* [Methods for Metagenomics](https://cloud.tsinghua.edu.cn/d/9553a9a553304ff7b311/?p=%2Fmethod%20-%20metagenomics&mode=list): tumor locator \(tissue match\), ranking, etc
* [Methods for Tumor Location](https://cloud.tsinghua.edu.cn/d/9553a9a553304ff7b311/?p=%2Fmethod%20-%20locator&mode=list): tumor locator \(tissue match\), etc

### 3.5 Transfer Learning

* 2020 _Bioinformatics_ - Exploiting transfer learning for the reconstruction of the human gene regulatory network
* \(see another example of transfer learning in SPOT-RNA [above](ai.md#3-1-rna-secondary-structure-prediction)\)

[PDFs](https://cloud.tsinghua.edu.cn/d/9553a9a553304ff7b311/?p=%2FTransfer%20Learning&mode=list)



