---
description: Literature Recommendations
---

# RNA Regulation

> * [PDFs for RNA Regulation](https://cloud.tsinghua.edu.cn/d/759f79f0a9c24fb7aab4/)

## 1. Noncoding RNA & Cancer

========================================================

* 2020 Nature - PCAWG - Analyses of non-coding somatic drivers in 2,658 cancer whole genomes

[PDFs](https://cloud.tsinghua.edu.cn/d/759f79f0a9c24fb7aab4/?p=%2FNoncoding%20RNA%20and%20cancer&mode=list)

## 2. Quantum Biology

========================================================

* 2014 Nature Physics - Quantum coherence in photosynthesis for efficient solar-energy conversion

[PDFs](https://cloud.tsinghua.edu.cn/d/759f79f0a9c24fb7aab4/?p=%2FQuantum%20Biology&mode=list)

## 3. RNA Regulation - POSTAR

**POSTAR: Post-transcriptional Regulation**

========================================================

### 3.1 RBP Binding: CLIP-seq etc

Data about RBP-RNA interaction

**Reviews**

* 2018 Nature Reviews - A brave new world of RNA-binding proteins
* 2018 Mol Cell - Advances in CLIP Technologies for Studies of Protein-RNA Interactions.pdf
* 2018 Annual Reviews - Data Science Issues in Studying Protein–RNA Interactions with CLIP Technologies 

**Research**

* Rbfox proteins regulate alternative mRNA splicing through evolutionarily conserved RNA bridges. \(Nature structural & molecular biology, 2013\)
* Identification of high-confidence RNA regulatory elements by combinatorial classification of RNA-protein binding sites. \(Genome Biology, 2017\)

[PDFs](https://cloud.tsinghua.edu.cn/d/759f79f0a9c24fb7aab4/?p=%2FRBP-RNA%20interaction&mode=list)

### 3.2 AS: Alternative splicing

**Reviews**

* Alternative splicing as a regulator of development and tissue identity. \(Nature Reviews, 2017\)
* RNA mis-splicing in disease. \(Nature Reviews, 2015\)

**Research**

* Transcriptome-wide isoform-level dysregulation in ASD, schizophrenia, and bipolar disorder. \(Science, 2018\)
* An Arabidopsis PWI and RRM motif-containing protein is critical for pre-mRNA splicing and ABA responses. \(Nature communications, 2015\)

#### Splicing & Cancer

* 2020 - _Nature Reviews_ - Roles and mechanisms of alternative splicing in cancer — implications for care

[PDFs](https://cloud.tsinghua.edu.cn/d/759f79f0a9c24fb7aab4/?p=/Splice%20and%20Cancer&mode=list)

### 3.3 APA: Alternative polyadenylation

**Reviews**

* Alternative polyadenylation of mRNA precursors. \(Nature Reviews, 2017\)
* Alternative cleavage and polyadenylation: extent, regulation and function. \(Nature Reviews, 2013\)

**Research**

* Dynamic analysis of alternative polyadenylation from RNA-seq reveal a 30-UTR landscape across seven tumour types. \(Nature communications, 2014\)
* Cellular stress alters 3′UTR landscape through alternative polyadenylation and isoform-specific degradation. \(Nature communications, 2018\)

[PDFs](https://cloud.tsinghua.edu.cn/d/759f79f0a9c24fb7aab4/?p=%2FAlternative%20polyadenylation&mode=list)

### 3.4 RNA Velocity

* _**2020 - Nature Biotech.**_ - Generalizing RNA velocity to transient cell states through dynamical modeling

[PDFs](https://cloud.tsinghua.edu.cn/d/759f79f0a9c24fb7aab4/?p=%2FRNA%20Velocity&mode=list)

### 3.5 RNA editing

**Review**

* A-to-I RNA editing — immune protector and transcriptome diversifier. \(Nature Reviews, 2018\)
* RNA editing-dependent epitranscriptome diversity in cancer stem cells. \(Nature Reviews, 2017\)

**Research**

* Systematic characterization of A-to-I RNA editing hotspots in microRNAs across human cancers. \(Genome Research, 2017\) 
* The Genomic Landscape and Clinical Relevance of A-to-I RNA Editing in Human Cancers. \(Cancer cell, 2015\)

[PDFs](https://cloud.tsinghua.edu.cn/d/759f79f0a9c24fb7aab4/?p=%2FRNA%20editing&mode=list)

### 3.6 DNA mutation using RNA-seq

* RNA sequence analysis reveals macroscopic somatic clonal expansion across normal tissues \(Science, 2019\) 

### 3.7 Translation: Ribo-seq

关于RNA翻译（Ribo-seq）

* ORF finding 
  * [RiboCode](https://www.ncbi.nlm.nih.gov/pubmed/29538776): 2018 _NAR_
  * [RiboWave](https://www.ncbi.nlm.nih.gov/pubmed/29945224): 2018 _NAR_
  * [RiboTaper](https://www.ncbi.nlm.nih.gov/pubmed/26657557): 2016 _Nature Methods_
* Difference of Translation Efficiency
  * [RiboDiff](https://www.ncbi.nlm.nih.gov/pubmed/27634950): 2017 _Bioinformatics_ 
  * [Xtail](https://pubmed.ncbi.nlm.nih.gov/27041671/): 2016 _Nature Communications_

| Preprocessing | ORF Iden. | Quantification TE | Translation elongation | Integrative & visualization |
| :--- | :--- | :--- | :--- | :--- |
| Rfoot | DeepRibo | scikit-ribo | diricore | Plastid |
| RiboWaltz | FLOSS | **RiboDiff** | Iχnos | Rfeet |
| **RiboWave** | ORF-RATER | riborex | PausePred | RiboGalaxy |
|  | PRICE | \*\*\*\*[**Xtail**](https://pubmed.ncbi.nlm.nih.gov/27041671/)\*\*\*\* | ROSE | RiboProfiling |
|  | RiboCode |  | RiboMiner | riboSeqR |
|  | RiboHMM |  |  | RiboTools |
|  | RibORF |  |  | RIVIT |
|  | **RiboTaper** |  |  | Shoelaces |
|  | RP-BP |  |  | tRanslatome |
|  | SPECtre |  |  | \*\*\*\*[**POSTAR**](http://lulab.life.tsinghua.edu.cn/postar/)\*\*\*\* |
|  | TITER |  |  |  |

[PDFs and PPTs](https://cloud.tsinghua.edu.cn/d/759f79f0a9c24fb7aab4/?p=/Riboseq&mode=list)

## 4. RNA Structure - Experimental Methods 

========================================================

### 4.1 Structurome

关于RNA 二级结构高通量测序方法

[PDFs](https://cloud.tsinghua.edu.cn/d/759f79f0a9c24fb7aab4/?p=/RNA%20Probing%20Experiments&mode=list)

> See computational models and algorithms for RNA secondary structure in [RNA Structure](ai.md#3-rna-structure).

### 4.2 RiboShape: Ribo-seq+SHAPE

* **2018** _**Cell**_  **\(e. coli, proposed a model\)** _****_- Pervasive regulatory functions of mRNA structure revealed by high-resolution SHAPE probing
* _**2014 NAR**_ **\(proposed a model\)** - Deciphering the rules by which dynamics of mRNA secondary structure affect translation efficiency in Saccharomyces cerevisiae
* **2018** _**Mol. Cell**_ **\(e. coli\)-** A Stress Response that Monitors and Regulates mRNA Structure Is Central to Cold Shock Adaptation
* **2020** _**Genome Biology**_ **\(zebrafish\)** _****_- RNA structural dynamics regulate early embryogenesis through controlling transcriptome fate and function
* _**2017 Nature**_ \(**plant**\)- Global translational reprogramming is a fundamental layer of immune regulation in plants

[PDFs](https://cloud.tsinghua.edu.cn/d/759f79f0a9c24fb7aab4/?p=/Riboshape&mode=list)



## 5. RNA Structure - Computational Methods

========================================================

> * **Course**: [生物物理学（清华大学）](https://www.ncrnalab.org/courses/#biophysics) 
> * **Book**:   《Biological Sequence Analysis: **Probabilistic Models** of Proteins and Nucleic Acids》by _Richard Durbin, Sean R. Eddy, Anders Krogh, Graeme Mitchison_  \([English](http://www.amazon.com/Biological-Sequence-Analysis-Probabilistic-Proteins/dp/0521629713) \| [中文](http://www.amazon.cn/dp/B003ZUIRZ2)\)

### 5.1 RNA Secondary Structure Prediction

* **Energy Model** 
  * **RNAstructure/Mfold**  and **RNAfold** 
    * How do RNA folding algorithms work? \(Sean R Eddy\) [2004 _Nature Biotech_](https://www.nature.com/articles/nbt1104-1457) 
    * What is dynamic programming? \(Sean R Eddy\) [2004 _Nature Biotech_](http://dx.doi.org/10.1038/nbt0704-909)\_\_
  * **SuperFold** for long distance base pairs \(folding\)  [2014 _Nature Methods_](https://www.ncbi.nlm.nih.gov/pubmed/25028896)\_\_

> **RNAstructure**/**Mfold**  and **RNAfold** perform good for sequence less than 200nt.
>
> **SuperFold** uses partition in RNAstructure package to predict partition functions for subsequences of long RNA, then merge the results. Therefore, it claims to perform better on long distance base pairs.

* **SCFG Model** \(Rfam/Infernal\)
  * What is a hidden Markov model? \(Sean R Eddy\) [2004 _Nature Biotech_](http://dx.doi.org/10.1038/nbt1004-1315) 
  * _You can read the SCFG section in the book_ [_above_](ai.md#3-1-rna-secondary-structure-prediction)_. \(Need a short tutorial for SCFG.\)_
* **Deep Learning Method**
  * \*\*\*\*[2020 ICLR](https://openreview.net/forum?id=S1eALyrYDH) - RNA Secondary Structure Prediction By Learning Unrolled Algorithms \([Chinese comments](https://mp.weixin.qq.com/s/SSFOJfljhRZuOOTErNefig)\)
  * \(**Transfer learning**\) 2019 _Nature Commn._ - SPOT-RNA: RNA secondary structure prediction using an ensemble of two-dimensional deep neural networks and transfer learning

[PDFs & PPTs](https://cloud.tsinghua.edu.cn/d/9553a9a553304ff7b311/?p=%2FRNA%20secondary%20structure%20prediction&mode=list)

### 5.2 Structural Motif Finder

* **SCFG \( Rfam/Infernal** [https://rfam.xfam.org/](https://rfam.xfam.org/) \)
  * **SCFG**: CMfinder - [2006 _Bioinformatics_](https://www.ncbi.nlm.nih.gov/pubmed/16357030) 
  * **SCFG**: RNApromo - [2008 _PNAS_ ](https://www.ncbi.nlm.nih.gov/pubmed/18815376)
  * **CFG:** TEISER - [2012 _Nature_](https://www.ncbi.nlm.nih.gov/pubmed/22495308)  
* **Graph Kernel:** GraphProt - [2014 _Genome Biology_ ](https://www.ncbi.nlm.nih.gov/pubmed/24451197)
* **SHAPE+HMM**:  PATTERNA - [2018 _Genome Biology_ ](https://www.ncbi.nlm.nih.gov/pubmed/29495968)

[PDFs & PPTs](https://cloud.tsinghua.edu.cn/d/9553a9a553304ff7b311/?p=%2FRNA%20Structural%20Motif&mode=list) 

## 



